includes/class-suggestion-engine.php

```
@@ -0,0 +1,347 @@
+			// --- Gzip/Brotli compression — strict boolean check ---
+			// Never compare against 'Enabled' string; use native bool from Telemetry.
+			$compression   = (bool) ( $telemetry['gzip_brotli_compression'] ?? false );
+			$suggestions[] = self::make_boolean(
+				'gzip_brotli_compression',
+				true === $compression,
+				'enable_server_rules',
+				'Gzip/Brotli Compression'
+			);
+
+			// --- Cache-Control headers — strict boolean check ---
+			$cc            = (bool) ( $telemetry['cache_control_headers'] ?? false );
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $cc. Configured minimum length is 3. (undefined)

(ShortVariable)